perfkitbenchmarker/configs/benchmark_config_spec.py

```diff
@@ -203,6 +203,9 @@ def _GetOptionDecoderConstructions(cls):
             'default': spark_service.PROVIDER_MANAGED,
             'valid_values': [spark_service.PROVIDER_MANAGED,
                              spark_service.PKB_MANAGED]}),
+        'version': (option_decoders.EnumDecoder, {
+            'default': spark_service.SPARK_1_6_2,
```

Comment:
Spark runner upgraded to 1.6.3 - though it really shouldn't matter much, and would be compatible.